src/player.js

```diff
@@ -276,6 +278,28 @@ class Player extends FakeEventTarget {
   }
 
   /**
+   * @function _onPlay
+   * @return {void}
+   * @private
+   */
+  _onPlay(): void {
+    this._eventManager.unlisten(this, Html5Events.PLAY);
```

Comment:
I so hate falgs...... (player V2...)
but you are right.
we must pass an handler to unlisten to avoid  this mistake in future